app/static/js/admin/guest/explore/event-box-component.js

```diff
@@ -22,7 +22,12 @@ Vue.component('event-box', {
             return moment(this.event.start_time).format('ddd, MMM DD HH:mm A');
         },
         locationName: function () {
-            return this.event.location_name.split(",")[0];
+            if(this.event.location_name){
```

Comment:
I think you should also check for non-empty string. This checks only for the None. Non-empty and undefined should also be checked I think.